packages/core/src/theming/prebuilt-themes/qcca/components/scrollbar.scss

```
@@ -0,0 +1,17 @@
+@use 'sass:map';
+@use '../base';
+
+*::-webkit-scrollbar {
+    width: 6px;
+}
+
+*::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 5px #dae6f0;
+    border-radius: 10px;
+}
+
+*::-webkit-scrollbar-thumb {
+    background: #095797;
```

Même chose ici, la couleur est disponible dans la palette
```suggestion
    background: base.$blue-normal-piv;
```